keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandlerTests.kt

```diff
@@ -92,13 +92,14 @@ internal class ApplicationLoadBalancerHandlerTests : JUnit5Minutests {
     |    - us-east-1d
     |listeners:
     | - port: 80
-    |   protocol: HTTP
+    |   protocol: HTTPS
+    |   certificateArn: this-is-my-certificate
```

Comment:
I think i like this change better. there will be no change in the config, just a change on how we call orca so things work?

Reply:
Correct

Reply:
I would still like to figure out how to better specify a cert (if possible), but this should get us unblocked